dask-apps/cupy_sum_mpi.py

```diff
@@ -0,0 +1,91 @@
+"""
```

Comment:
Same question here

Reply:
This can be moved as well. 

@ZackWRyan